sdcm/wait.py

```diff
@@ -156,12 +156,14 @@ def wait_for_log_lines(node, start_line_patterns, end_line_patterns, start_timeo
         started = any(start_follower)
         while not started and (time.time() - start_time < start_timeout):
             started = any(start_follower)
+            time.sleep(0.05)
```

Comment:
I think that `0.05` is too small for this case.
Our time windows are measured mostly in minutes.

Reply:
doesn't make much difference - waking up 200 times a second is not a big deal.

Reply:
it's way better than the zero time it was sleeping before :)

Let's start with that, and we'll see if we need to adapt it further